packages/backend/server/src/core/workspaces/resolvers/doc.ts

```diff
@@ -266,6 +269,26 @@ export class WorkspaceDocResolver {
     return paginate(rows, 'createdAt', pagination, count);
   }
 
+  @ResolveField(() => PaginatedDocType, {
+    description: 'Get recently updated docs of a workspace',
+  })
+  async recentlyUpdatedDocs(
+    @CurrentUser() me: CurrentUser,
+    @Parent() workspace: WorkspaceType,
+    @Args('pagination', PaginationInput.decode) pagination: PaginationInput
+  ): Promise<PaginatedDocType> {
+    const [count, rows] = await this.models.doc.paginateDocInfoByUpdatedAt(
```

Comment:
this query is large, need to add a workspace read permission check first to prevent ddos

Reply:
should wait for https://github.com/toeverything/AFFiNE/pull/12861 merge